### PR TITLE
[feature] Snowflake: Generate ALL grants for 0.20.0

### PR DESCRIPTION
**Pull request by edulop91**
### Summary
Grab the v0.20.0 release of the snowflake provider and re-generate ALL grant modules with it. There are a handful of new modules being created through that process that recently got merged to the provider. 

### Test Plan
should have automatically generated integration + unit tests

### References

### CODE REVIEW

File: snowflake-external-table-grant-all/main.tf.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```
Comment (cziie) [inline snowflake-external-table-grant-all/main.tf.json:46]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-file-format-grant-all/main.tf.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```
Comment (cziie) [inline snowflake-file-format-grant-all/main.tf.json:41]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-function-grant-all/main.tf.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```
Comment (cziie) [inline snowflake-function-grant-all/main.tf.json:56]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-materialized-view-grant-all/main.tf.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```
Comment (cziie) [inline snowflake-materialized-view-grant-all/main.tf.json:46]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-procedure-grant-all/main.tf.json [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```
Comment (cziie) [inline snowflake-procedure-grant-all/main.tf.json:56]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-schema-grant-all/main.tf.json [hunk 6]
```diff
@@ -39,6 +39,7 @@
   },
   "locals": {
     "privileges": [
```
Comment (cziie) [inline snowflake-schema-grant-all/main.tf.json:41]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-sequence-grant-all/main.tf.json [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```
Comment (cziie) [inline snowflake-sequence-grant-all/main.tf.json:41]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-stream-grant-all/main.tf.json [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+      "default": null
+    },
+    "with_grant_option": {
+      "type": "bool",
+      "description": "When this is set to true, allows the recipient role to grant the privileges to other roles.",
+      "default": null
+    }
+  },
+  "locals": {
+    "privileges": [
```
Comment (cziie) [inline snowflake-stream-grant-all/main.tf.json:41]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-table-grant-all/main.tf.json [hunk 9]
```diff
@@ -46,6 +46,7 @@
     "privileges": [
```
Comment (cziie) [inline snowflake-table-grant-all/main.tf.json:46]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-view-grant-all/main.tf.json [hunk 10]
```diff
@@ -44,6 +44,7 @@
   },
   "locals": {
     "privileges": [
```
Comment (cziie) [inline snowflake-view-grant-all/main.tf.json:46]: ⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used

File: snowflake-external-table-grant-all/main.tf.json (hunk 1 above)
Comment (edulop91) [reply snowflake-external-table-grant-all/main.tf.json:46]: I don't think this is true. Going to try upgrading tflint versions

**Review by ryanking — APPROVED** [review]

**Pull request merged**